clock.rb

```diff
@@ -27,6 +27,10 @@ module Clockwork
     Clock::SubscriptionsBillerJob.perform_later
   end
 
+  every(1.day, 'schedule:finalize_invoices', at: '02:00') do
+    Clock::FinalizeInvoicesJob.perform_later
```

Comment:
We will probably have to run it every hour for the timezone feature, since everything is based on the status I think it will not be an issue.